### DISCUSSION TITLE
Problem when I zoom in an image using glightbox

### DISCUSSION THREAD

**Discussion by IvyMasterDiego** [squidfunk/mkdocs-material#4904] [Q&A] [answered]
When I click on the images, they don't load me correctly, I thought it was where the link was being obtained from but when I open those links, they don't load correctly, could you help me or something, here I put a screenshot what this looks like:

![imagen](https://user-images.githubusercontent.com/45220823/214056943-97becc23-0dc8-4539-b94f-e0502de52aa7.png)

**Comment by squidfunk** [comment, marked as answer]
Please report this to [`mkdocs-glightbox`](https://github.com/blueswen/mkdocs-glightbox).

**Reply by IvyMasterDiego** [reply]
oke
